packages/toolkit/src/query/tests/polling.test.tsx

```diff
@@ -3,6 +3,7 @@ import { createApi } from '@reduxjs/toolkit/query'
 import { setupApiStore, waitMs } from './helpers'
 import { delay } from '../../utils'
 import type { SubscriptionSelectors } from '../core/buildMiddleware/types'
+import { createListenerMiddleware } from '@reduxjs/toolkit'
```

Comment:
you don't need createListenerMiddleware for setupListeners 😛

Reply:
I see, I made my life harder than it had to be trying to set up a new store with it and apparently just didn't test to see if it was in the current storeRef in that test file... and then was busy with work for a day or two.

Anyway should reflect this change in most recent revision.